### ISSUE TITLE
Pip install is broken since v 1.3.5

### ISSUE THREAD

**Issue by MichaelScript** [plasticityai/magnitude#26]
Hi,

It seems like during the v1.3.5 release when you moved the installation files it broke pip installing

I've tried both in a docker container running debian jessie as well as on my mac.

Steps to reproduce:
```
# Breaks
pip install pymagnitude
pip install pymagnitude==0.1.35
```
```
# This works
pip install pymagnitude==0.1.34
```

Error:
```
Building wheels for collected packages: pymagnitude
  Running setup.py bdist_wheel for pymagnitude ... done
  Stored in directory: /Users/mycoolusername/Library/Caches/pip/wheels/d3/b0/69/5c2868a48835e8e79c2580c641548e8fdf953bfc29df4a
Successfully built pymagnitude
Could not install packages due to an EnvironmentError: [Errno 2] No such file or directory: '/usr/local/lib/python3.6/site-packages/numpy-1.15.0.dist-info/METADATA'
```

I believe the problem is with the python package file move, the solution would most likely to be to move it back to where it was originally or edit the package config file (I forget).

Thank you for making such an awesome project.

**Comment by AjayP13** [comment]
Confirmed this is an issue, looking into it, thanks for the report!

**Comment by AjayP13** [comment]
I believe it was caused by me trying to force an upgrade of the local `pip` in the installer. If your `pip` needed to be updated, it would fail the install as it was screwing with `pip`'s files as it was using them for the `pymagnitude` install. Once it was updated, it would succeed (which is why it works on the second try). I just removed the code for that requirement, the CI/CD runners are building that fix on 0.1.52, which I'll verify fixes it with `pip` once its out.

Thanks for the report!
